gameModel/libs/ClientScript/UIfacade/methaneFacade.ts

```diff
@@ -0,0 +1,38 @@
+import { hasContainerOfType } from '../game/common/resources/emergencyDepartment';
+import {
+  ResourceContainerType,
+  ResourceContainerTypeArray,
+  UniqueResourceTypeMap,
+} from '../game/common/resources/resourceContainer';
+import { getCurrentState } from '../game/mainSimulationLogic';
+import { getTypedInterfaceState } from '../gameInterface/interfaceState';
+
+export function containerTypeAvailable(type: ResourceContainerType): boolean {
```

Comment:
```suggestion
export function isContainerTypeAvailable(type: ResourceContainerType): boolean {
```
[personal preferences] : I like to have is / has at the start of a method returning a boolean. Do you too ?

Reply:
I can live with both